tests/unit-tests/formatting/functions.php

```diff
@@ -702,6 +706,21 @@ public function test_wc_timezone_string() {
 	 */
 	public function test_wc_timezone_offset() {
 		$this->assertEquals( 0.0, wc_timezone_offset() );
+
+		// Test with float GMT offset.
+		update_option( 'gmt_offset', 2.5 );
+		$this->assertEquals( 2.5 * 60 * 60, wc_timezone_offset() );
+
+		// Test with int GMT offset.
+		update_option( 'gmt_offset', 2 );
+		$this->assertEquals( 2 * 60 * 60, wc_timezone_offset() );
+
+		// Test with negative offset.
+		update_option( 'gmt_offset', -2 );
+		$this->assertEquals( -2 * 60 * 60, wc_timezone_offset() );
+
+		// Restore default.
+		update_option( 'gmt_offset', '0' );
```

Comment:
Is this necessary? I believe that all the database changes are reverted automatically after each test.

Reply:
Tested and it's necessary, also other tests on this file does the same.